tests/testthat/test-interactive_report.R

```diff
@@ -0,0 +1,39 @@
+test_that("peergroup correct",{
+
+  expect(peer_group %in% c("pensionfund", "assetmanager", "bank" ,"insurance"))
+
+})
+
+test_that("language valid",{
+
+  expect(language_select %in% c("EN", "DE", "FR"))
+
+})
+
+
+test_that("template folder exists for selected language",{
+
+  expect(dir.exists(paste0(template_path, template_name)))
```

Comment:
when this test runs, where should `template_path` and `template_name` be coming from?